vaccine_feed_ingest/runners/us/giscorps_vaccine_providers/normalize.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import os
+import pathlib
+import re
+import sys
+from typing import List, Optional
+
+from vaccine_feed_ingest_schema import location as schema
+
+from vaccine_feed_ingest.utils.log import getLogger
+
+logger = getLogger(__file__)
+
+
+SOURCE_NAME = "us_giscorps_vaccine_providers"
+
+
+def _get_availability(site: dict) -> schema.Availability:
+    appt_only = site["attributes"]["appt_only"]
+    if appt_only == "Yes":
+        return schema.Availability(appointments=True)
+    elif appt_only is not None:
+        return schema.Availability(drop_in=True)
+
+    return None
+
+
+def _get_id(site: dict) -> str:
+    data_id = site["attributes"]["GlobalID"]
+
+    # Could parse these from the input file name, but do not for now to avoid
+    # accidental mutation.
+    arcgis = "c50a1a352e944a66aed98e61952051ef"
+    layer = 0
+
+    return f"{arcgis}_{layer}_{data_id}"
+
+
+def _get_inventory(site: dict) -> Optional[List[schema.Vaccine]]:
+    # vaccines_field = site["attributes"]["flu_vaccinations"].lower().split(",")
+
+    # potentials = {
+    #     "pfizer": schema.Vaccine(vaccine="pfizer"),
+    #     "moderna": schema.Vaccine(vaccine="moderna"),
+    #     "janssen": schema.Vaccine(vaccine="janssen"),
+    #     "jjj": schema.Vaccine(vaccine="janssen"),
+    # }
+
+    # inventory = []
+
+    # for vf in vaccines_field:
+    #     try:
+    #         inventory.append(potentials[vf])
+    #     except KeyError as e:
+    #         logger.error("Unexpected vaccine type: %s", e)
+
+    # if len(inventory) > 0:
+    #     return inventory
```

Comment:
Do we plan to capture this in the future? If not, let's remove the method!

Reply:
i dont believe thsi dataset ever had the intention of including this data. will remove